src/plugins/js-eval/jsEvalPlugin.js

```diff
@@ -59,7 +59,7 @@ const jsEvalPlugin = async ({ mentionUser, respond, message, selfConfig = {} })
     );
     respond((mentionUser ? `${mentionUser}, ` : '(okay) ') + result);
   } catch (e) {
-    respond((mentionUser ? `${mentionUser}, ` : '') + e); // Error message always start with Error:
+    respond((mentionUser ? `${mentionUser}, ` : '(fail) ') + e.message); // Error message always start with Error:
```

Comment:
@ljharb yes I agree, but those Errors come from jsEval.js, they are less important to showthan the actual run error, in short it currently outputs: '(jsEval)Error: (run)Error: message'

I'm thinking about doing a `reject(data)` [here](https://github.com/brigand/jellobot/blob/e2717ecd344369d52c7e7a0f8b628b92e684ac94/src/plugins/js-eval/jsEval.js#L41) but that's not a good idea because other places [reject with an Error instance](https://github.com/brigand/jellobot/blob/e2717ecd344369d52c7e7a0f8b628b92e684ac94/src/plugins/js-eval/jsEval.js#L35)

Reply:
Yep, that's what it does, I added a test case